### ISSUE TITLE
16.7.1 Still can't launch some Android apks

### ISSUE THREAD

**Issue by switchy85** [koying/SPMC#992]
Using 16.7.1 aarch64 launcher version on a shield TV running 3.3 (Android 6.0). Had the exact same issue on 16.7.
Only certain apks won't open, and they're seemingly ones that were sideloaded, with a couple exceptions. Netflix, Hulu, YouTube, xplore,  and Play Store all open fine. Droidmote server and home switcher won't open and I'm pretty sure that they came from the play store. Amazon video, mobdro, xposed, chrome, and an app that goes to the shield settings just won't open at all. And a few apps like terrarium tv and Google play music open up an app selection box that lists multiple versions of the same app and sometimes gives errors when opening.

**Comment by switchy85** [comment]
Forgot to post up the log. Booted up and launched Netflix fine, then tried to launch Amazon video and nothing happened, then tried to launch terrarium tv and it gave the selection box thing.

**Comment by switchy85** [comment]
http://paste.semperpax.com/pjnlfdvrh

**Comment by koying** [comment]
It's fixed in 17.
I might backport it to 16, not sure of the motivation, yet...

**Comment by switchy85** [comment]
I hope you do. I'd use v17 but I use it as a launcher and that version hasn't been released yet.

**Comment by esoteradactyl** [comment]
Yeah a backport would be great. I just have no love for krypton. 16.7.1 is where it's at 
;)
